DashAI/front/src/types/plugin.ts

```diff
@@ -0,0 +1,19 @@
+import type { PluginStatus } from "../pages/plugins/constants/pluginStatus";
+
+interface Tag {
+  id: number;
+  name: string;
+  plugin_id: number;
+}
+export interface IPlugin {
+  id: number;
+  name: string;
+  author: string;
+  tags: Tag[];
```

Comment:
Since you already declared the tags as inteface, wouldn't it be a good idea that the tag array also be of type tag?
```suggestion
  tags: Array<ITag>];
```